pkgs/desktops/mate/pluma/default.nix

```diff
@@ -47,6 +48,11 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  postFixup = ''
+    buildPythonPath ${python3.pkgs.six}
```

Comment:
Nice catch :+1: . I still got lots of `Couldn't find foreign struct converter for 'cairo.Context'` though, last time I saw `Gtk.cairo_*` usages I just added `python3.pkgs.pycairo`, not sure if that applies here, but anyway I can at least enable the plugin now